frontend/src/components/filter/FilterMenu.js

```diff
@@ -126,8 +125,7 @@ export default function FilterMenu({ filters, onApplyFilters, applyButtonAria })
 
   const onUpdateFilter = (id, name, value) => {
     //
-    // this is bonkers... we need to do more than map the array
-    // what escaped me originally was that just an array spread creates a new
+    // just an array spread creates a new
```

Comment:
Do we want to keep some of the comment above?

Reply:
I'll remove it